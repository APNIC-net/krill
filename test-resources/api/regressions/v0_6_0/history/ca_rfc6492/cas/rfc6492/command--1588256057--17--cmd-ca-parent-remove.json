{
  "actor": "krill",
  "time": "2020-04-30T14:14:17.417942Z",
  "handle": "rfc6492",
  "version": 22,
  "sequence": 17,
  "details": {
    "remove_parent": "ta"
  },
  "effect": {
    "events": [
      22
    ]
  }
}