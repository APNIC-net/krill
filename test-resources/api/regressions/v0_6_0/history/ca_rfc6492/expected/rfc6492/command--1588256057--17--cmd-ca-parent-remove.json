{
  "command": {
    "actor": "krill",
    "time": "2020-04-30T14:14:17.417942Z",
    "handle": "rfc6492",
    "version": 22,
    "sequence": 17,
    "details": {
      "remove_parent": "ta"
    },
    "effect": {
      "events": [
        22
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "rfc6492",
        "version": 22,
        "details": {
          "parent_removed": [
            "ta",
            [
              {
                "ca_repo": "rsync://localhost/repo/rfc6492/1",
                "added": [],
                "updated": [],
                "withdrawn": [
                  {
                    "name": "9F29B373BC1B3D173E815841AE492E3E64CC6325.mft",
                    "hash": "8d3e8caf2c4fab6beeb5886a8343a31af6e3a12dca2acb24c6c7cbdd8a480727"
                  },
                  {
                    "name": "9F29B373BC1B3D173E815841AE492E3E64CC6325.crl",
                    "hash": "67abb7e409efd75f43f3ac419eef9cd6129b9548ad8ef0c101a02d8e5ab3d558"
                  }
                ]
              }
            ]
          ]
        }
      }
    ]
  }
}